modules/local/collect_featurecounts.nf

```diff
@@ -40,7 +40,7 @@ process COLLECT_FEATURECOUNTS {
                         lazy_dt() %>%
                         filter(count > 0) %>%
                         mutate(
-                            sample = str_remove(sample, 'task.sort.bam'),
+                            sample = str_remove(sample, '.sorted.bam.bam'),
```

Comment:
Should this really be "bam.bam"? Not just ".bam"?

Reply:
I checked and the output and it is .sorted.bam.bam that's why it has this str_remove